interface/usergroup/user_admin.php

```diff
@@ -228,11 +228,28 @@ function authorized_clicked() {
 <body class="body_top">
 
 <div class="container">
-
+    <?php
+    /*  Get the list ACL for the user */
+    $is_super_user = acl_check('admin', 'super');
+    $acl_name=acl_get_group_titles($iter["username"]);
+    $bg_name='';
+    $bg_count=count($acl_name);
+    $selected_user_is_superuser = false;
+    for ($i=0; $i<$bg_count; $i++) {
+        if ($acl_name[$i] == "Emergency Login") {
+            $bg_name=$acl_name[$i];
+        }
+        //check if user member on group with superuser rule
+        if (!$user_is_superuser && is_group_include_superuser($acl_name[$i])) {
```

Comment:
Do you mean $selected_user_is_superuser instead of $user_is_superuser?

Reply:
fixed..thanks.